src/sources.rs

```diff
@@ -193,6 +206,7 @@ pub fn iter(source_dir: &Path, report: &Report) -> io::Result<Iter> {
         dir_deque: dir_deque,
         report: report.clone(),
         last_apath: None,
+        excludes: excludes.clone()
```

Comment:
(minor) Probably reasonable to just pass `excludes` by value and consume it, rather than cloning.